src/components/Main/Modal/UserModal/UserModal.vue

```diff
@@ -136,12 +132,13 @@ export default defineComponent({
 }
 
 .icon {
+  @include background-secondary;
```

Comment:
```suggestion
  background-color: $theme-background-secondary;
```
中で使ってる`background-size`とかを上書きしてしまうっぽい